kombu/serialization.py

```diff
@@ -57,9 +57,14 @@ def _reraise_errors(wrapper,
 def pickle_loads(s, load=pickle_load):
     # used to support buffer objects
     try:
-        return load(BytesIO(s))
+        f = BytesIO(s)
+        return load(f)
     except UnicodeDecodeError:
-        return load(BytesIO(s), encoding='latin1')
+        f = BytesIO(s)
+        return load(f, encoding='latin1')
+    except TypeError as exc:
+        print(s)
```

Comment:
Is this going to make it to our log stream properly?

Reply:
I think stdout will get redirected to syslog, otherwise using a `logger` we also need to think about changing the webapp logger config which will filter out any non-webapp loggers that are not explicitly registered.

Reply:
I would want to check with our logging friends to verify before we go through the trouble of deploying this. Also, I think we should base64 encode it to avoid any issues in transit.

Reply:
Added b64 encoding, staging envs indicate `print` should work